### PR TITLE
Ouerea Nation Cape

### PR DESCRIPTION
**Pull request by RustingWithYou**
Due to the planet's Skrell population, adds an Ouerea flag cape to the Skrell xenowear loadout.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

File: html/changelogs/RustingWithYou - ouereacape.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: ChangeMe
```
Comment (alsoandanswer) [inline html/changelogs/RustingWithYou - ouereacape.yml:30]: yea

File: html/changelogs/RustingWithYou - ouereacape.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Added a changelog editing system that should cause fewer conflicts and more accurate timestamps."
+  - rscdel: "Killed innocent kittens."
```
Comment (DreamySkrell) [inline html/changelogs/RustingWithYou - ouereacape.yml:42]: 💀

**Review by DreamySkrell — CHANGES_REQUESTED** [review]

File: html/changelogs/RustingWithYou - ouereacape.yml (hunk 2 above)
Comment (RustingWithYou) [reply html/changelogs/RustingWithYou - ouereacape.yml:42]: this is what i get for doing 2 prs within 5 minutes of each other

**Review by DreamySkrell — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**